### DISCUSSION TITLE
.ipa app for ios - I have a developer certificate to sign ipa -

### DISCUSSION THREAD

**Discussion by fercesa** [KRTirtho/spotube#966] [General]
Is there a .ipa app to be able to sign it on my iPhone? 
I have a developer certificate to sign ipa in IOS

**Comment by powenn** [comment]
Try https://github.com/powenn/spotube/releases/tag/3.3.0
Should work for TrollStore and sideloadly

**Comment by KRTirtho** [comment]
Well, it can work without signing like (@powenn was able to do), but the certificate is required to publish to App Store and in that case I'll need my own certificate.
But no worries with the help of OpenCollective fund we'll be able to publish the app to AppStore as soon as we get our hands on a mac. I am guessing in the 1st quarter of 2024 we can release for iOS (if App Store approves)

**Comment by fercesa** [comment]
Thanks a lot!

I use sideloadly to Install it in my iPhone,
The app is brilliant, I found only one trouble, I can’t play music while using another app and when I put my phone to sleep


Enviado desde Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: powen ***@***.***>
Enviado: sábado, diciembre 30, 2023 5:48 a. m.
Para: KRTirtho/spotube ***@***.***>
CC: fercesa ***@***.***>; Author ***@***.***>
Asunto: Re: [KRTirtho/spotube] .ipa app for ios - I have a developer certificate to sign ipa - (Discussion #966)


Try https://github.com/powenn/spotube/releases/tag/3.3.0
Should work for TrollStore and sideloadly
